src/main/java/me/botsko/prism/database/mysql/MySqlPrismDataSource.java

```diff
@@ -14,9 +15,7 @@
 import java.io.FileOutputStream;
 import java.io.IOException;
 import java.io.OutputStream;
-import java.sql.PreparedStatement;
-import java.sql.ResultSet;
-import java.sql.SQLException;
+import java.sql.*;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Using the '.*' form of import should be avoided - java.sql.*.](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5472183)